config/routes.rb

```diff
@@ -77,7 +77,11 @@
         json_api_resources :pages, controller: "project_pages"
       end
 
-      json_api_resources :workflows, links: [:subject_sets], versioned: true
+      json_api_resources :workflows, links: [:subject_sets], versioned: true do
+        member do
+          post :retire_subject
```

Comment:
I'm open to suggestions for a better location, it feels slightly wacky where it is now but I can't think of anything better.